### ISSUE TITLE
Training time?

### ISSUE THREAD

**Issue by Wenchao-Du** [pkuxmq/Invertible-Image-Rescaling#32]
How much time do you train the model in your paper? Thank you

**Comment by pkuxmq** [comment]
Hi, I trained models on one Tesla P100 or one Tesla P40 GPU, and it takes about 6 days for the 500k iterations in the paper.

**Comment by Wenchao-Du** [comment]
thanks for your quick reply, this is a nice work, but it is truly for normalized flow models that cost more time to get good results than GANs and VAEs.

**Comment by pkuxmq** [comment]
500k iterations are not too long for image rescaling tasks. The common setting for image super-resolution models (e.g. RCAN, ESRGAN) is to train 1000k iterations, and ESRGAN will take another 400k iteration for fine-tuning with GAN. Considering we would compute both downscaling and upscaling through INN, we reduced the iterations by half, and the overall training time should be about the same level as these models.

**Comment by Wenchao-Du** [comment]
ok. recently i have tried the INN for other restoration task and found it is slower to get pleasent results than other GAN models, and i also found related work, e.g., Glow, also costs more time to generate visual friendly results. Anyway, thank you.

**Comment by yang-Jin-hai** [comment]
> 

Hi, in the paper I think you described that you train the model for **50k** iterations in total (rather than **500k**), and the learning rate decays at 10k, 20k, 30k, 40k iters.
It somehow confuse me, since it is different from the codes and related discussions.
Is it a typo in the paper? I think the real num of total iterations may be 500k.

**Comment by yang-Jin-hai** [comment]
> Hi, I trained models on one Tesla P100 or one Tesla P40 GPU, and it takes about 6 days for the 500k iterations in the paper.

Hi @pkuxmq, in the paper I think you described that you train the model for 50k iterations in total (rather than 500k), and the learning rate decays at 10k, 20k, 30k, 40k iters.
It somehow confuses me, since it is different from the codes and related discussions.
Is it a typo in the paper? I think the real num of total iterations may be 500k.
